components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -1252,7 +1254,7 @@ private LocalClaim createLocalClaim(LocalClaimReqDTO localClaimReqDTO) {
         claimProperties.put(PROP_REQUIRED, String.valueOf(localClaimReqDTO.getRequired()));
         claimProperties.put(PROP_SUPPORTED_BY_DEFAULT, String.valueOf(localClaimReqDTO.getSupportedByDefault()));
         if (localClaimReqDTO.getDataType() != null) {
-            claimProperties.put(PROP_DATA_TYPE, String.valueOf(localClaimReqDTO.getDataType()));
+            claimProperties.put(PROP_DATA_TYPE, String.valueOf(localClaimReqDTO.getDataType()).toLowerCase());
```

Comment:
This change is required to avoid the data types being inserted in upper case. 
The enums are defined in upper case letters but the actual value should be inserted in lower case as the previous data types were defined in lower case
ex: string, date_time, integer